evol/population.py

```diff
@@ -43,7 +45,8 @@ def __init__(self,
                  generation: int=0,
                  intended_size: Optional[int]=None,
                  checkpoint_target: Optional[str]=None,
-                 serializer=None):
+                 serializer=None,
+                 concurrent_workers: int=1):
```

Comment:
Since `Pool` also accepts `processes` to be `None`, shouldn't we make this `Optional[int]`?